lambda/delete-user-services/delete-user-services.ts

```diff
@@ -1,3 +1,64 @@
-const handler = async (): Promise<void> => console.log("Delete function");
+import { SNSEvent } from "aws-lambda";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import {
+  DynamoDBDocumentClient,
+  DeleteCommand,
+  DeleteCommandOutput,
+} from "@aws-sdk/lib-dynamodb";
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { UserData } from "./models";
 
-export default handler;
+const { TABLE_NAME } = process.env;
```

Comment:
We may as well combine this with 
```
const { DLQ_URL } = process.env;
```
on line 28 (I'm surprised there's not a ESLint rule to catch this).

Reply:
Good spot, done!